shared/README.md

```diff
@@ -1,3 +1,21 @@
 # shared
 
 Shared component between the frontend and backend, e.g. shared password validation.
+
+## Development
+
+In order to build ESM and CommonJS versions of the module in its directory run:
+
+```bash
+yarn build
+```
+
+...or from the project's root directory:
+
+```bash
+yarn workspace vaa-shared build
+```
+
+Currently ESM version is used by `vaa-frontend` and CommonJS by `backend/vaa-strapi` modules.
```

Comment:
Out of curiosity 😅 . Why does the backend/vaa-strapi project use CommonJS?

Reply:
Good question! We use this `strapi-plugin-import-export-entries` plugin, which is a CommonJS module. So we are kinda stack with having to support CommonJS over ESM. 

Internally Strapi devs chose to stick with CommonJS for reasons unknown, they replied in a GitHub thread that they decided it is not worth switching (years ago). :(